### PR TITLE
Adds yet another FBP, also fixes a little sprite error

### PR DESCRIPTION
**Pull request by SubberTheFabulous**
Nobody even uses those, why do I bother?
Preview:
image

### CODE REVIEW

File: code/modules/organs/robolimbs.dm [hunk 1]
```diff
@@ -97,9 +97,15 @@ var/const/standard_monitor_styles = "blank=ipc_blank;\
...
 	company = "Cyber Solutions - Array"
 	desc = "This limb is simple and functional; array of sensors on a featureless case."
-	icon = 'icons/mob/human_races/cyberlimbs/cybersolutions/cybersolutions_alt1.dmi'
+	icon = 'icons/mob/human_races/cyberlimbs/cybersolutions/cybersolutions_alt2.dmi'
 	unavailable_to_build = 1
 	parts = list(BP_HEAD)
+	
+/datum/robolimb/cybersolutions_alt1
+	company = "Cyber Solutions - Wight"
+	desc = "This limb has cheap plastic panels, mounted on grey metal."
```
Comment (Belsima) [inline code/modules/organs/robolimbs.dm]: No need for a comma.

File: code/modules/organs/robolimbs.dm [hunk 2]
```diff
@@ -97,9 +97,15 @@ var/const/standard_monitor_styles = "blank=ipc_blank;\
 /datum/robolimb/cybersolutions_alt1
```
Comment (Anewbe) [inline code/modules/organs/robolimbs.dm]: This path should be changed, otherwise the other one will never show up.

**Comment by SubberTheFabulous** [comment]
_Now_, everything should make sense.

**Comment by Keekenox** [comment]
I use FPBs. I appreciate you.

**Pull request merged**